### PR TITLE
Poster Server log parsing functionality added

### PR DESCRIPTION
**Pull request by EgorAfutin**

### CODE REVIEW

File: src/test/unit/applications/log_parsing/poster_server/poster_server_log_parsing.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+            },
+            {
+                '@timestamp': datetime(2017, 9, 4, 12, 30, 33, 740000),
+                'level': 'ERROR',
+                'module': 'ResizerModule',
+                'message': 'File not found (EventImages/1.jpg)'
+            }
+        )
+
+    def test_parse_fails(self):
```
Comment (needmorec) [inline src/test/unit/applications/log_parsing/poster_server/poster_server_log_parsing.py]: please, remove this test with have particular for data parsing

File: src/applications/log_parsing/poster_server/driver.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+from util.utils import Utils
+
+
+def create_event_creators(configuration=None):
+    """
+    Tree of different parsers for all types of logs for poster server
+    :param configuration: YML config
+    :return: Tree of event_creators
+    """
+    poster_server_log = EventCreator(Metadata([TimestampField("@timestamp", "%Y-%m-%d %H:%M:%S.%f"),
```
Comment (vunder-kind) [inline src/applications/log_parsing/poster_server/driver.py]: We should use ConfigurableTimestampField for setting timezone for timestamps

File: src/applications/log_parsing/poster_server/driver.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+                                               StringField("level"),
+                                               StringField("module"),
+                                               StringField("message")]),
+                                     RegexpParser(
+                                         r"^(?P<timestamp>\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2}\.\d{3})"
```
Comment (vunder-kind) [inline src/applications/log_parsing/poster_server/driver.py]: In requirements we have comma between seconds and milliseconds

File: src/test/unit/applications/log_parsing/poster_server/poster_server_log_parsing.py [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+
+class PosterServerMessageParsingTestCase(BaseMultipleMessageParsingTestCase):
+    event_creators = create_event_creators()
+
+    def test_parse_poster_server_log(self):
+        self.assert_parsing(
+            {
+                'source': 'PosterServer.log',
+                'message': '2017-09-04 12:30:33.740   WARN    Config OverlaysDirectory not set'
```
Comment (vunder-kind) [inline src/test/unit/applications/log_parsing/poster_server/poster_server_log_parsing.py]: In requirements we have comma between seconds and milliseconds

**Pull request merged**